index.html

```diff
@@ -1,1642 +1,264 @@
+      </p>
+      <p>
+        The following list contains the handlers event types, supported by all MiniApp component elements:  
+      </p>
+      <dl>
+        <dt>`touchstart`</dt>
+        <dd>Like the <a href="https://www.w3.org/TR/touch-events/#dfn-touchstart" data-link-type="dfn">touchstart</a> event type defined in [[TOUCH-EVENTS]], this event is dispatched on the topmost component (event target) indicated by the pointer when the user places a <a data-link-type="dfn" href="https://w3c.github.io/touch-events/#dfn-touch-point">touch point</a> on the touch surface.</dd>
+        <dt>`touchmove`</dt>
+        <dd>Like the <a href="https://www.w3.org/TR/touch-events/#dfn-touchmove" data-link-type="dfn">touchmove</a> event type defined in [[TOUCH-EVENTS]], this event is dispatched on the topmost component (event target) indicated by the pointer when the user moves a <a data-link-type="dfn" href="https://w3c.github.io/touch-events/#dfn-touch-point">touch point</a> along the touch surface.</dd>
+        <dt>`touchcancel`</dt>
+        <dd>Like the <a href="https://www.w3.org/TR/touch-events/#dfn-touchcancel" data-link-type="dfn">touchcancel</a> event type defined in [[TOUCH-EVENTS]], this event is dispatched on the topmost component (event target) indicated by the pointer when the <a data-link-type="dfn" href="https://w3c.github.io/touch-events/#dfn-touch-point">touch point</a> has been disrupted in an implementation-specific manner.</dd>
+        <dt>`touchend`</dt>
+        <dd>Like the <a href="https://www.w3.org/TR/touch-events/#dfn-touchend" data-link-type="dfn">touchend</a> event type defined in [[TOUCH-EVENTS]], this event is  dispatched on the topmost component (event target) indicated by the pointer when the user removes a <a data-link-type="dfn" href="https://w3c.github.io/touch-events/#dfn-touch-point">touch point</a> from the touch surface, also including cases where the touch point physically leaves the touch surface.</dd>
+        <dt>`click` or `tap`</dt>
+        <dd>Like the <a href="https://www.w3.org/TR/uievents/#event-type-click" data-link-type="dfn">click</a></th> event type defined in [[UIEVENTS]], this event is dispatched on the topmost event target indicated by the pointer when the user presses down and releases the primary pointer button or otherwise activates the pointer in a manner that simulates such an action.</dd>
+        <dt>`longtap` or `longpress`</dt>
+        <dd>The `longpress` or `longtap` event type is dispatched on the topmost component (event target) indicated by the pointer when the user presses down, holds for at least (<mark>@@@ms@@@</mark>), and releases the primary pointer button or otherwise activates the pointer in a manner that simulates such an action.</dd>
+      </dl>
+      <div class="ednote">
+        Could we define the minimum period to be considered as a long press?

```

Comment:
I think we could define the minimum period.  
As a developer, I want to know how long it is.
If possible, maybe we can provide a configuration that allows developers to modify the default duration in the future.

Reply:
I haven't find the durations in the specific implementations. Should we indicate that can be configured by the platform and developer?

Reply:
I've added a note with the value that Android uses for this event. As reference, but not mandatory. Please have a look: 89f116779ab7cdd286a1a846e1786b03234d0615